cpp/include/cudf/round_robin.hpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2018-2019, NVIDIA CORPORATION.
```

Comment:
`(c) 2020`? Dunno if this has unexpected legal implications.

Reply:
Technically, work on this started in 2019. I can change it, if it's okay. Is it okay?

Reply:
Done.